### DISCUSSION TITLE
WIFI Module and the External Sound Module

### DISCUSSION THREAD

**Discussion by Mr-JimmyDeath** [UzixLS/zx-sizif-512#42] [Q&A] [answered]
Hello

I have found that if you plug in the Sizif external sound card and then try and use the WiFI module it does not respond unless you first enter the Magic Menu and set 'TSFM+MIDI' to off

Is this expected behaviour?

Thanks

James

**Comment by UzixLS** [comment, marked as answer]
Yes, wifi module is connected via AY on the main board, which is disabled when TSFM active.

**Reply by Mr-JimmyDeath** [reply]
Thank you

Regards

James
